setup/rest-api-docs.md

```diff
@@ -177,7 +177,7 @@ Response:
     "qid": "my_question_id1",
     "aid": "my_answer_id1",
     "time": <TimeFormatTBD>, 
-    "responses-left": <int>
```

Comment:
you think responses_left is better cause of consistency? like we have my_question_id

Reply:
All underscores you see are in values not keys. "my_question_id1" is just a placeholder for whatever the id will end up being.

There will be no snake case - the default for javascript is camelcase.